### PR TITLE
release

### PR DESCRIPTION
**Pull request by tinosulzer**

### CODE REVIEW

**Comment by priyanshuone6** [comment]
Does pypi automatically remove 0 from the version as v21.08 is v21.8 in pypi

**Comment by tinosulzer** [comment]
I guess so

**Comment by priyanshuone6** [comment]
If that's the case then we should match the name of this release with pypi

File: pybamm/version [hunk 1]
```diff
@@ -1 +1 @@
-21, 8
+21, 9
```
Comment (priyanshuone6) [inline pybamm/version:1]: Or maybe this should be 21, 09

Comment (brosaplanella) [reply pybamm/version:1]: I think it is read as an `int` and the formatted when it is printed. About PyPi, I think I saw some warning in the actions (probably the one that builds the wheels) that said that PyPi would ignore the 0, so we might as well remove it...

**Review by brosaplanella — APPROVED** [review]

**Comment by priyanshuone6** [comment]
@tinosulzer You reverted `pybamm/version` back to 21, 8 in the latest commit

**Pull request merged**